rskj-core/src/main/java/co/rsk/core/RskAddress.java

```diff
@@ -69,7 +70,7 @@ public RskAddress(String address) {
      */
     public RskAddress(byte[] bytes) {
         if (bytes.length != LENGTH_IN_BYTES) {
-            throw new RuntimeException(String.format("An RSK address must be %d bytes long", LENGTH_IN_BYTES));
+            throw new RpcInvalidRskAddressException(String.format("An RSK address must be %d bytes long", LENGTH_IN_BYTES));
```

Comment:
`RskJsonRpcRequestException` is an `JSON-RPC` layer error, but `RskAddress` is a core class that's being used in multiple layers. I wouldn't introduce this kind of exception in this class

Reply:
Addressed